docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md

```diff
@@ -75,8 +75,11 @@ Following is a typical \*.runsettings file. Each element of the file is optional
   
           </CodeCoverage>  
         </Configuration>  
-      </DataCollector>  
+      </DataCollector>
   
+      <!--Video data collector is only available with Visual Studio 2017 Update 5 and higher -->
```

Comment:
What does update 5 refer to exactly? Is there a specific version we can use instead, e.g. Visual Studio 2017 version 15.6?